webapp/graphql/User.js

```diff
@@ -283,3 +283,12 @@ export const currentUserQuery = gql`
     }
   }
 `
+
+export const currentUserCountQuery = () => gql`
+  ${userCountsFragment}
+  query {
+    currentUser {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-27T11:05:15Z" title="Wednesday, May 27th 2020, 1:05:15 pm +02:00">May 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -265,9 +265,11 @@ export const checkSlugAvailableQuery = gql`
 
 export const currentUserQuery = gql`
   ${userFragment}
+  ${userCountsFragment}
   query {
     currentUser {
       ...user
+      ...userCounts
```


I think you have forgotten to delete this?
This will cause performance problems …

🤔 As I deleted this line the delete account page can not be entered anymore …
… so here seems to be a real problem in my eyes.

I investigated this and may you have a look at the users profile page `webapp/pages/profile/_id/_slug.vue` and how itÄs done there.
Use `UserQuery` and `User` as variable. See
`import UserQuery from '~/graphql/User‘`

Instead of `currentUser` and variable `this.currentUserCount`.
Then you can even get totally rid of `currentUserCountQuery`.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-07-30T11:40:52Z" title="Thursday, July 30th 2020, 1:40:52 pm +02:00">Jul 30, 2020</time>_

---


ok